js/packages/redux/reducers/chatInputs.reducer.ts

```diff
@@ -12,15 +12,23 @@ const makeRoot = <T>(val: T) => ({
 	[sliceName]: val,
 })
 
+export type ChatInputSelectionType = { start: number; end: number } | null
+
 type ChatInputState = {
 	id: string
 	text: string
+	selection: ChatInputSelectionType
+	isFocused: boolean
+	isSending: boolean
 	mediaList: string[] // cids
 }
 
 const newChatInputState: (convPK: string) => ChatInputState = convPK => ({
 	id: convPK,
 	text: '',
+	selection: null,
+	isFocused: false,
+	isSending: false,
```

Comment:
are you sure you want to persist state that will not be in sync after an app restartt